test/analytics-manager.ts

```diff
@@ -173,22 +175,59 @@ describe('Redux Analytics Manager - Functionality', function() {
       chai.expect(spies.sendSpy.calledOnce).to.be.false;
     }
   );
+
+  it('allows users to de-register an action', async function() {
+    const sendSpy = spies.sendSpy;
+    await this.store.dispatch(Util.actionCreator1());
+    chai.expect(sendSpy.calledOnce).to.be.true;
+    manager.deRegisterAction(Util.ACTION1);
+    await this.store.dispatch(Util.actionCreator1());
+    chai.expect(sendSpy.calledOnce).to.be.true;
```

Comment:
I think calledOnce resolves to true, even if it was called more than once. I think we might want to do something like

`chai.expect(sendSpy.callCount).to.equal(1);`

Reply:
good point! updating...

Reply:
updated